pkg/liveshare/session.go

```diff
@@ -123,6 +123,20 @@ func (s *Session) StartJupyterServer(ctx context.Context) (int, string, error) {
 	return port, response.ServerUrl, nil
 }
 
+func (s *Session) Rebuild(ctx context.Context) error {
```

Comment:
nit: `RebuildContainer` - `session.Rebuild` describes to me that the liveshare session itself will be rebuilt/reconnected, etc.